cg/services/delivery_message/messages/rna_fastq_scout_message.py

```diff
@@ -0,0 +1,60 @@
+from cg.services.delivery_message.messages.delivery_message import DeliveryMessage
+from cg.services.delivery_message.messages.utils import (
+    get_caesar_delivery_path,
+    get_scout_link,
+)
+from cg.store.models import Case
+from cg.store.store import Store
+
+
+class RNAFastqScoutMessage(DeliveryMessage):
+    def __init__(self, store):
+        super().__init__()
+        self.store = store
+
+    def create_message(self, cases: list[Case]) -> str:
+        if len(cases) == 1:
+            return self.get_case_message(cases[0])
+        return self.get_cases_message(cases)
+
+    def get_case_message(self, case: Case) -> str:
+
+        related_uploaded_dna_cases: list[Case] = self.store.get_uploaded_related_dna_cases(
+            rna_case=case
+        )
+        scout_links: list[str] = [get_scout_link(case) for case in related_uploaded_dna_cases]
+        scout_links_row_separated: str = "\n".join(scout_links)
```

Comment:
This was done because all the other scout delivery messages had this structure. Do you think I should refactor all of them in this PR? Or just add this `_get_scout_links_row_separated` function only for the messages I introduce here and leave the others as they are?

Reply:
If you extract it into a utils method, it makes sense to fix it for the other scout delivery messages as well. Not the end of the world but it would be a nice touch

Reply:
💯